modules/control-utility/src/test/java/org/apache/ignite/util/GridCommandHandlerTest.java

```diff
@@ -752,6 +755,34 @@ public void testIdleVerifyOnInactiveClusterWithPersistence() throws Exception {
         assertContains(log, testOut.toString(), "The check procedure has finished, no conflicts have been found.");
     }
 
+    /**
+     *
+     */
+    @Test
+    public void testIdleVerifyCancelCommand() throws Exception {
+        IgniteEx srv = startGrids(4);
+
+        srv.cluster().state(ACTIVE);
+
+        IgniteCache<Integer, Integer> cache =  srv.createCache(DEFAULT_CACHE_NAME);
+
+        for (int i = 0; i < 100000; i++)
+            cache.put(i, i);
+
+        new Thread(() -> execute("--cache", "idle_verify") ).start();
```

Comment:
`--cache idle_verify` don't return while command ends?
If yes, we need to do the following:
1. Start with `GridTestUtils.runAsync` which returns Future.
2. Check future not done.
3. Cancel command.
4. Check future done.
5. Other checks for all servers.

Reply:
refactored

Reply:
> 4. Check future done.

AFAICS this not implemented.